Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'activeadmin', github: 'activeadmin'
 
 # for handling file uploads
 gem 'carrierwave'
+gem 'fog', require: false
```

Comment:
Please add proper comment why 'fog' is needed. I think it's because carrierwave uses 'fog' to  handle s3 uploads. If that's the case then mention that.